src/core/client_channel/subchannel.cc

```diff
@@ -697,8 +698,9 @@ void Subchannel::OnRetryTimer() {
 
 void Subchannel::OnRetryTimerLocked() {
   if (shutdown_) return;
-  gpr_log(GPR_INFO, "subchannel %p %s: backoff delay elapsed, reporting IDLE",
-          this, key_.ToString().c_str());
+  gpr_log(GPR_INFO,
```

Comment:
This does not seem like a good change.  The subchannel key is useful to log here, and the pid and tid are not.  Please revert this change.